SCons/Tool/msvs.xml

```diff
@@ -39,70 +41,106 @@ See its __doc__ string for a discussion of the format.
   <builder name="MSVSProject">
     <summary>
       <para>
-        Builds a Microsoft Visual Studio project file, and by default
-        builds a solution file as well.
+        Build a Microsoft Visual C++ project file and solution file.
       </para>
       <para>
-        This builds a Visual Studio project file, based on the
-        version of Visual Studio that is configured (either the
-        latest installed version, or the version specified by
-        &cv-link-MSVS_VERSION; in the Environment constructor). For
-        Visual Studio 6, it will generate a <filename>.dsp</filename>
-        file. For Visual Studio 7, 8, and 9, it will
-        generate a <filename>.vcproj</filename> file.  For Visual
-        Studio 10 and later, it will generate a
-        <filename>.vcxproj</filename> file.
-      </para>
-      <para>
-        By default, this also generates a solution file for the
-        specified project, a <filename>.dsw</filename> file for
-        Visual Studio 6 or a <filename>.sln</filename> file for
-        Visual Studio 7 and later. This behavior may be disabled by
-        specifying <literal>auto_build_solution=0</literal> when you
-        call &b-MSVSProject;, in which case you presumably want to
-        build the solution file(s) by calling the &b-MSVSSolution;
-        Builder (see below).
-      </para>
-      <para>
-        The &b-MSVSProject; builder takes several lists of filenames
-        to be placed into the project file. These are currently
-        limited to <literal>srcs</literal>, <literal>incs</literal>,
-        <literal>localincs</literal>, <literal>resources</literal>, and
-        <literal>misc</literal>. These are pretty self-explanatory,
-        but it should be noted that these lists are added to the
-        &cv-link-SOURCES; construction variable as strings, NOT as
-        SCons File Nodes.  This is because they represent file names
-        to be added to the project file, not the source files used
-        to build the project file.
-      </para>
-      <para>
-        The above filename lists are all optional, although at least
-        one must be specified for the resulting project file to
+        Builds a C++ project file based on the
+        version of Visual Studio (or to be more precise, of MSBuild)
+        that is configured: either the latest installed version,
+        or the version specified by
+        &cv-link-MSVC_VERSION; in the current &consenv;.
```

Comment:
Not MSVS_VERSION? That's what I see in msvs.py..?

Reply:
It's been long deprecated, although it still copies over MSVC_VERSION to MSVS_VERSION so they're both set.  I figured we should be consistent in what we say.

Reply:
But given that now there's not necessarily any correlation, wouldn't using MSVS_VERSION be potentially more correct going forward?
If say MSVS 2023 changes the project file format, but you're using MSVC_VERSION 2017..?

Reply:
I did bring that up earlier, but I'm less convinced now it's an issue.  It's not really a Visual Studio issue: MSBuild is the driver, and since they went to the XML format it's been stable for 12+ years now. And if they do change it, we won't know the new format .... (for a while, anyway)

Reply:
They do seem to be getting better at documenting some of this type of stuff. But I'm sure we'll find out from a user since maintainers aren't consuming this currently..